pkg/forwarder/forwarder.go

```diff
@@ -387,58 +343,50 @@ func (f *DefaultForwarder) sendHTTPTransactions(transactions []*HTTPTransaction)
 // SubmitSeries will send a series type payload to Datadog backend.
 func (f *DefaultForwarder) SubmitSeries(payload Payloads, extra http.Header) error {
 	transactions := f.createHTTPTransactions(seriesEndpoint, payload, false, extra)
-	transactionsSeries.Add(1)
```

Comment:
Why removing this line?

Reply:
It's removed because in `createHTTPTransactions`, I add things into the map `transactionsCount` with the endpoint type as the `key`, so there is no need to have an explicit `transactionXXX.Add` anymore.